src/core/structures/acroform/CheckBox.ts

```diff
@@ -0,0 +1,46 @@
+import { PDFArray, PDFDict, PDFName } from 'src/core';
+import { PDFAcroButton } from './index';
+
+class CheckBox extends PDFAcroButton {
```

Comment:
Can we name this class `PDFAcroCheckBox`? 

I also think we should rename `PushButton` to `PDFAcroPushButton` (I missed it in my review): 

https://github.com/Hopding/pdf-lib/blob/1ee2f4cd2329a21a9821e54f6a9b0a8b516d0d6b/src/core/structures/acroform/PushButton.ts#L4

Reply:
Updated